packages/skeleton-svelte/src/components/tags-input/anatomy/item-delete-trigger.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts" module>
+	import type { HTMLAttributes } from '@/internal/html-attributes';
+	import type { PropsWithElement } from '@/internal/props-with-element';
+
+	export interface TagsInputItemDeleteTriggerProps extends PropsWithElement<'button'>, HTMLAttributes<'button'> {}
+</script>
+
+<script lang="ts">
+	import { classesTagsInput } from '@skeletonlabs/skeleton-common';
+	import { mergeProps } from '@zag-js/svelte';
+
+	import X from '@/internal/components/x.svelte';
+
+	import { TagsInputItemContext } from '../modules/item-context';
+	import { TagsInputRootContext } from '../modules/root-context';
+
+	const props: TagsInputItemDeleteTriggerProps = $props();
+
+	const tagsinput = TagsInputRootContext.consume();
+	const itemProps = TagsInputItemContext.consume();
+
+	const { element, children = x, ...rest } = $derived(props);
+
+	const attributes = $derived(
+		mergeProps(tagsinput().getItemDeleteTriggerProps(itemProps()), rest, {
+			class: classesTagsInput.itemDeleteTrigger,
+		}),
+	);
+</script>
+
+{#snippet x()}
+	<X class="size-4" />
```

Comment:
Consider replacing the Icon with:

```html
<strong>&times;</strong>
```

This is how we [handled it in v3](https://github.com/skeletonlabs/skeleton/blob/v3/packages/skeleton-svelte/src/components/TagsInput/TagsInput.svelte#L81) and it creates a smaller/bolder X symbol that looks a bit nicer IMO:

<img width="355" height="114" alt="Screenshot 2025-09-15 at 11 04 48 AM" src="https://github.com/user-attachments/assets/0f8d2aa3-5ed2-4372-a267-dddf423d1154" />

I wouldn't mind this for the FileUpload file "close" button icons too. Either way we should probably settle on a convention here.